src/browser/web-preview.js

```diff
@@ -66,13 +69,16 @@
       borderRadius: '4px',
       boxShadow: '0 1px 3px rgba(0, 0, 0, 0.4)',
       color: '#444',
-      float: 'left',
       height: '300px',
       margin: '0 10px',
       overflow: 'hidden',
       position: 'relative',
-      'scrollSnapCoordinate': '50% 50%',
-      width: '240px'
+      width: '240px',
+      top: '50%',
+      marginTop: '-50%',
+      // Seems to make things worse
+      // transition: 'ease-in opacity 0.1s'
+      transition: 'ease box-shadow 0.15s'
```

Comment:
Animations are usually `<property> <time> <easing function>` but CSS is liberal in what it accepts, so this works.